app/main.py

```diff
@@ -1,8 +1,38 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str)\
+            -> None:
```

Comment:
Jak mniemam flake8 zwrócił uwagę, ze pojedyncza linia jest zbyt długa. Bardziej czytelnie byłoby zrzutować do takiej postaci:

```suggestion
    def __init__(
        self, 
        comfort_class: int, 
        clean_mark: int,
        brand: str
     ) -> None:
```